src/routes/(infoPages)/[topTierPage]/[...serviceGroupPage]/+page.svelte

```diff
@@ -30,11 +30,28 @@
     // Call To Actions within service entry accordions are rich text,
     // leaving no way to designate additional formatting/styling for them.
     // We must manually add the required classes to achieve the desired styles.
+    // While we're at it, attach an analytics event so we can track CTA clicks.
     // This wasn't always getting applied in `onMount`, so we're making sure it doesn't
     // happen prematurely by using `afterUpdate` instead.
-    document
-      .querySelectorAll(".service-entry-CTA p > a")
-      .forEach((linkEl) => linkEl.classList.add("usa-button"));
+    document.querySelectorAll(".usa-accordion__heading").forEach((accordionHeading) => {
+      const serviceAccordionHeadingText = accordionHeading.textContent;
+      const serviceAccordionCTA = accordionHeading.nextElementSibling?.querySelector(
+        ".service-entry-CTA p > a",
+      );
+      if (serviceAccordionCTA) {
+        serviceAccordionCTA.classList.add("usa-button");
+        serviceAccordionCTA.addEventListener("click", () => {
+          if (window.gtag) {
+            window.gtag("event", "ldaf_service_cta_click", {
+              event_category: "click",
+              event_label: serviceAccordionCTA.textContent,
+              ldaf_service_name: serviceAccordionHeadingText,
+              ldaf_service_cta_destination: serviceAccordionCTA.getAttribute("href"),
+            });
+          }
```

Comment:
I was originally thinking about breaking out an analytics "send event" function into a separate service, but I think it's simple enough that we don't need to do this. Only real reason to break it out would be to add retry logic, which I don't think is necessary (at least not yet).